content/en/tracing/faq/php-tracer-manual-installation.md

```diff
@@ -75,45 +75,111 @@ The correct value for `PHP_EXTENSION_VERSION` depends on the PHP version.
 
 ### Install from source
 
-[Download the source code `tar.gz` or `.zip` file][3] from the releases page and unzip the file. Then compile and install the extension with the commands below.
+Installation from source is only recommended if the provided installers do not work in a specific platform. For example on arm64 machines.
 
-```bash
-$ cd /path/to/dd-trace-php
-$ phpize
-$ ./configure --enable-ddtrace
-$ make
-$ sudo make install
-```
-
-#### Modify the INI file
-
-Modify the `php.ini` configuration file to make the **ddtrace** extension available in the PHP runtime. To find out where the INI file is, run the following command:
+#### Centos example
 
-```bash
-$ php --ini
+Follow the instructions below to build the PHP extension in a Centos container. Paths might vary depending on the PHP installation method, but the `Dockerfile` below should be easily adaptable to any environment.
 
-Configuration File (php.ini) Path: /usr/local/etc/php/7.2
-Loaded Configuration File:         /usr/local/etc/php/7.2/php.ini
-...
 ```
-
-Add the following line to the `php.ini` file.
-
-```ini
-extension=ddtrace.so
+FROM centos:8
```

Comment:
I think we already discussed this in slack, but it would be easier to use this page if you break this code example out into steps and then smaller code examples for each step